addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -70,7 +70,7 @@
                             <small t-if="attachment.extension" class="text-uppercase" t-esc="attachment.extension"/>
                         </div>
                     </t>
-                    <div class="o-mail-AttachmentCard-aside position-relative rounded-end overflow-hidden" t-att-class="{ 'o-hasMultipleActions d-flex flex-column': attachment.isDeletable and !env.inComposer }">
+                    <div class="o-mail-AttachmentCard-aside position-relative rounded-end overflow-hidden" t-att-class="{ 'o-hasMultipleActions d-flex flex-column': showDelete and !env.inComposer }">
```

Comment:
Note: this part is being fixed separately in https://github.com/odoo/odoo/pull/165605
As it is blocking on our prod, I think they wanted to move faster?

But we can keep the current PR for actually fixing the getter + adding test.

Reply:
Yes I saw that move by, thanks for the heads up.

This fix is still useful as updating the template won't work without updating the app.